README.rst

```diff
@@ -247,6 +247,47 @@ in terms of disk space.
 
 Whether ping checks are created automatically for devices.
 
+``OPENWISP_MONITORING_AUTO_CONFIG_MODIFIED``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``bool``    |
++--------------+-------------+
+| **default**: | ``True``    |
++--------------+-------------+
+
+Whether ``config_modified`` checks are created automatically for devices.
```

Comment:
This should be explained in a better way, since `config_modified check` doesn't say much at all to newcomers

Reply:
I was trying to be consistent with the `ping` check but yes you are right. I shall work upon improving the documentation :smiley: